src/HttpHeader.cc

```diff
@@ -486,6 +488,15 @@ HttpHeader::parse(const char *header_start, size_t hdrLen, Http::ContentLengthIn
             return 0;
         }
 
+        const bool frameHeader = (e->id == Http::HdrType::CONTENT_LENGTH || e->id == Http::HdrType::TRANSFER_ENCODING);
+        if (seenLines > 1 && frameHeader) {
```

Comment:
I would optimize this a little (at the expense of code simplicity) because the vast majority of header fields are not about framing and only have one line:

```C++
if (seenLines > 1) {
    const auto framing = ...
    if (framing) {
        ...
    }
}

Reply:
Done in 6ae264393

Reply:
> Done in 6ae264393

No such commit in the pushed branch. The last commit is currently 1a826d5.

Reply:
Odd. It is showing up for me - page refresh?

Reply:
Resolving. FWIW, I see nothing odd about it: I wrote my comment at HH:12. GitHub [event](https://github.com/squid-cache/squid/settings/hooks/14998733) `cb151f00-d5d9-11ea-88da-e10f90b7ced5` announced the push of your commit 6ae2643 about 20 minutes later, around HH:36. FWIW, the corresponding GitHub push event `1f7fa800-d4cf-11ea-92e2-c3899794ada9` for the previous commit (1a826d5) went out more than a day earlier.